pkg/virt-handler/vm.go

```diff
@@ -152,6 +153,13 @@ type VirtualMachineController struct {
 	podIsolationDetector     isolation.PodIsolationDetector
 	containerDiskMounter     *container_disk.Mounter
 	clusterConfig            *virtconfig.ClusterConfig
+
+	// records if pod network phase1 has completed
+	// phase1 involves cycling an entire posix thread
+	// so for performance, knowing phase1 is complete
+	// prevents cycling an unncessary posix thread.
+	phase1NetworkSetupCache     map[types.UID]int
```

Comment:
maybe use a Boolean or an empty struct instead?

Reply:
i stored the "pid" as the maps value. I cross reference the pid id just to be certain the setup has actually been done. In theory this pid cross examination shouldn't be necessary, i'm just not 100% sure what happens in the event of several failed migrations... there might be a edge case in there where an entry exists for a vmi in the cache from a previous migration attempt.

Reply:
thank you for clarifying this